eth/tracers/api.go

```diff
@@ -1027,3 +1043,54 @@ func overrideConfig(original *params.ChainConfig, override *params.ChainConfig)
 
 	return copy, canon
 }
+
+// argsToTransaction produces a Transaction object given call arguments.
+// The `msg` field must be converted from the same arguments.
+func argsToTransaction(b Backend, args *ethapi.TransactionArgs, msg *core.Message) (*types.Transaction, error) {
```

Comment:
as the msg was calculated from `args`, and we did have `args.toTransaction` in https://github.com/ethereum/go-ethereum/blob/2274a03e339f213361453590b54917bbfd0a0c31/internal/ethapi/transaction_args.go#L284

So I'd prefer to use `args.ToTransaction` instead.